tests/fixtures/end-tag-without-matching-open-element/multiple/expected/x/component/component.html

```diff
@@ -0,0 +1,2 @@
+<template>
+    <span>hello </span></template>
```

Comment:
Same thing here, there isn't a straightforward way to reformat the tags to line up correctly.